topics/jenkins/index.md

```diff
@@ -0,0 +1,13 @@
+---
+created_by: Kohsuke Kawaguchi
+display_name: Jenkins
+github_url: https://github.com/jenkinsci/jenkins
```

Comment:
Should it be a GitHub organization link actually?
E.g. https://github.com/git is used in https://github.com/topics/git

Reply:
others linked directly to main repo, but it was a tossup. :)

Perhaps better to link to https://github.com/jenkinsci due to pinned repos.

Reply:
```suggestion
github_url: https://github.com/jenkinsci
```